esp-hal/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Fixed input/output for GPIO touch pins (#1956)
```

Comment:
I would probably instead add the number of this PR to the touch support's line, I don't think it's worth calling out fixing something that wasn't broken in a prior release.